File: AkitaModel/Species.cs
namespace Skyware.Lis.AkitaModel;

/// <summary>
/// Species, e.g. Human, Dog, Cat, etc.
/// </summary>
public class Species
{

    /// <summary>
    /// Primary key. Value 1 is reserved for "Human".
    /// </summary>
    public int Id { get; set; }

    /// <summary>
    /// Name of the species, e.g. "Human".
    /// </summary>
    public string Name { get; set; }

}

File: AkitaModelJava/akitamodel/src/main/java/skyware/lis/akitamodel/Species.java
package skyware.lis.akitamodel;

import lombok.Data;

/**
 * Species, e.g., Human, Dog, Cat, etc.
 */
@Data
public class Species {

    /**
     * Primary key. Value 1 is reserved for "Human".
     */
    private int id;

    /**
     * Name of the species, e.g., "Human".
     */
    private String name;
    
}

